tests/performance-test/Dockerfile

```diff
@@ -14,7 +14,9 @@ FROM tripleomaster/centos-binary-collectd:current-tripleo-rdo
 USER root
 
 RUN yum install golang -y && \
-    yum update -y && \
+    #yum update -y && \  # skipping this step due to issue with
```

Comment:
Is it the full `yum update` that causes an issue? Can you just run the `yum update-minimal --security -y` instead?

I'm not overly worried about this since clearly this is just localized testing, so feel free to skip over.